ere-guests/stateless-validator/reth/risc0/Cargo.toml

```diff
@@ -18,7 +18,7 @@ reth-primitives-traits = { workspace = true, features = [
     "serde-bincode-compat",
 ] }
 
-revm = { workspace = true, features = ["std", "blst", "bn"] }
+revm = { workspace = true, features = ["std", "c-kzg", "blst", "bn"] }
```

Comment:
Unrelated to this PR: just noticed was missing in the recent https://github.com/eth-act/zkevm-benchmark-workload/pull/212 change.